src/lib/holocene/api-pagination.svelte

```diff
@@ -179,7 +179,7 @@
     </div>
     <nav
       class="flex flex-col justify-end gap-4 md:flex-row"
-      aria-label={$$restProps['aria-label']}
+      aria-label="{$$restProps['aria-label']} 1"
```

Comment:
Maybe "top" and "bottom" would be better here?

Reply:
I was trying to keep it something that wouldn't need to be translated (hence the 1 and 2) 😅